velox/vector/tests/VectorTestBase.cpp

```diff
@@ -58,6 +57,19 @@ BufferPtr VectorTestBase::makeIndices(
   return indices;
 }
 
+BufferPtr VectorTestBase::makeIndices(
+    const std::vector<vector_size_t>& indices) const {
+  auto size = indices.size();
+  BufferPtr indicesBuffer =
+      AlignedBuffer::allocate<vector_size_t>(size, pool());
+  auto rawIndices = indicesBuffer->asMutable<vector_size_t>();
+
+  for (int i = 0; i < size; i++) {
```

Comment:
I think you can use memcpy or std::copy  something like : 
`memcpy (rawIndices, indices.data(), indicies.size () * sizeof(vector_size_t)); `